src/Messages/Interactions/TSMessage.m

```diff
@@ -132,6 +132,12 @@ - (nullable instancetype)initWithCoder:(NSCoder *)coder
         _attachmentIds = [NSMutableArray new];
     }
 
+    if (!_receivedAtDate) {
+        // TSIncomingMessage.receivedAt has been superceded by TSMessage.receivedAtDate.
```

Comment:
If this is going to be a no-op for every class except TSIncomingMessage, maybe this belongs in TSIncomingMessage